lib/screens/home/cubit/profile_cubit.dart

```diff
@@ -3,19 +3,19 @@ import 'package:ecellapp/core/res/errors.dart';
 import 'package:ecellapp/core/res/strings.dart';
 import 'package:ecellapp/core/utils/logger.dart';
 import 'package:ecellapp/models/user.dart';
-import 'package:ecellapp/screens/home/tabs/profile/profile_repository.dart';
+import 'package:ecellapp/screens/home/home_repository.dart';
 import 'package:equatable/equatable.dart';
 
 part 'profile_state.dart';
 
 class ProfileCubit extends Cubit<ProfileState> {
-  final ProfileRepository _profileRepository;
-  ProfileCubit(this._profileRepository) : super(ProfileLoading());
+  final HomeRepository _homeRepository;
+  ProfileCubit(this._homeRepository) : super(ProfileLoading());
 
   Future<void> profile(String token) async {
```

Comment:
🔴 
function name should always be a verb. use `getProfile` or similar.

Reply:
done